Dockerfile

```diff
@@ -34,4 +34,5 @@ COPY . .
 ENV VERSION "$VERSION"
 ENV RESOURCES_URL "THIS_WILL_BE_REPLACED_BY_RUNTIME_ENV_VARIABLE"
 ENV HOST "{{hostname}}"
+RUN ./node_modules/.bin/lerna run build --scope @opencrvs/commons
```

Comment:
Do we need this actually? Because we run `yarn build` after this i.e. `lerna run build --stream --concurrency=1`

Reply:
Do some of the packages require this to be built before building them? I thought this is mostly to ensure this is built first

Reply:
Good point. But then again how was this working before as it wasn't there previously right?

Reply:
It was part of package.json in `compose:push:version` (removed in this PR)

```
"compose:push:version": "lerna run build --scope @opencrvs/commons &&
```

Reply:
Even in here I think it was redundant because like after running `compose:push:version`, two versions of the same image gets uploaded to the dockerhub, one with the `commit` tag and the other with `latest` tag. And the ones with the `latest` tag actually gets uploaded before the script even executes `lerna run build --scope @opencrvs/commons`.

Reply:
I think you are right 👍   I removed the part now completely